test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -85,6 +89,7 @@
     private static final String BOOKKEEPER_IMAGE_NAME = System.getProperty("bookkeeperImageName", "bookkeeper");
     private static final String TIER2_NFS = "nfs";
     private static final String TIER2_TYPE = System.getProperty("tier2Type", TIER2_NFS);
+    private static final boolean LATER_OPERATOR = getOperatorVersion();
```

Comment:
Should this field be named as `OPERATOR_VERSION`?

Reply:
Thats a boolean field indicating if the operator version is >= 0.4.0 or lower than that.
Yes, we could do with a better name here. 
Perhaps `IS_OPERATOR_VERSION_ABOVE_0_4_0`

Reply:
done